src/core/agent/AgentService.ts

```diff
@@ -21,18 +26,24 @@ export default class AgentService {
     #agent?: Agent
     #agentLanguage?: Language
     #pubsub: PubSub
+    #tokens: Map<string, AuthInfo>
```

Comment:
Should the name of this map be `#requests`? It is not storing the actual capability tokens, or is it?

Reply:
`requests` sounds better, it stores the `requestid-randnumber` -> `auth(capabilities, desc)`.